### ISSUE TITLE
Black squares

### ISSUE THREAD

**Issue by kffwlp** [luckydog7/Funkin-android#31]
There are black squares instead of characters.

**Comment by kffwlp** [comment]
I think the texture is glitched

**Comment by RMH240** [comment]
yeah someone need to fix this

**Comment by N201FNF** [comment]
I was trying to fix it but I couldn't

**Comment by aminos-ze-best** [comment]
I think this only works with the bf, but you need to press the back button and go to charting mode, and go to the second tab and you'll see "bf", you need to change it to "bf-car".

**Comment by aminos-ze-best** [comment]
Press the back button while you're playing a song.

**Comment by RMH240** [comment]
> Press the back button while you're playing a song.

I know and thats annoying.
like is there any normal way to do this?
and not using debug mode?

**Comment by silent-ui** [comment]
yes i know how to fix this already link:https://youtu.be/l_4PZo-8TpY

**Comment by kffwlp** [comment]
Thank you, do you have the updated version?

**Comment by kffwlp** [comment]
> yes i know how to fix this already link:https://youtu.be/l_4PZo-8TpY

Thank you, do you have the updated version?

**Comment by AndreFireOff2** [comment]
Man Its just put in 4092 resulation
